### ISSUE TITLE
Reconnect on close

### ISSUE THREAD

**Issue by bradabelgit** [apollographql/subscriptions-transport-ws#110]
Is there currently a way to completely disconnect a client? If I call the close method, a close event is fired off which prompts the client to attempt to reconnect... which seems wrong if I am explicitly asking to close the connection. I'd like to make sure I am completely cleaning things up when a user logs out. Am I missing something here?

I am doing the following when a user logs out.

```
shutdown() {
    this.wsClient.unsubscribeAll();
    this.wsClient.close();
    delete this.wsClient;
  }
```

Thanks :)

**Comment by Urigo** [comment]
@bradabelgit 
Does https://github.com/apollographql/subscriptions-transport-ws/pull/137 fixes it?

**Comment by mistic** [comment]
@bradabelgit I found the same problem because I have the same use case as you even before I found this issue :D There was a pull request to solve this already merged into master #137 :)

**Comment by bradabelgit** [comment]
@Urigo @mistic Thanks guys! Will update the package and confirm over the weekend 👍

**Comment by mistic** [comment]
@bradabelgit please be aware that u have to use the master branch to test or wait for the next v0.0.7 release that is about to coming out :D

**Comment by dotansimha** [comment]
Merged and available in 0.7.0. thanks @bradabelgit and @mistic !
